File: uri/media_tres/media_tres.kt
import java.util.Scanner
import kotlin.text.*

val pesoA : Double = 2.0
val pesoB : Double = 3.0
val pesoC : Double = 5.0

fun media( notaA : Double, notaB : Double, notaC : Double ) : Double {

    return ((notaA * pesoA) + (notaB * pesoB) + (notaC * pesoC)) / (pesoA + pesoB + pesoC);
}

fun main(args: Array<String>){
    var input = Scanner(System.`in`)
    var notaA : Double
    notaA = (input.nextLine()).toDouble()

    var notaB : Double
    notaB = (input.nextLine()).toDouble()

    var notaC : Double
    notaC = (input.nextLine()).toDouble()

    var resposta : Double    
    resposta = media(notaA, notaB, notaC)
    println("MEDIA = %.1f".format(resposta))
}

File: uri/media_tres/media_tres.py
def media( notaA, notaB, notaC ):
    return ((notaA * 2.0) + (notaB * 3.0)+ (notaC * 5.0)) / 10.0

notaA = float(input())
notaB = float(input())
notaC = float(input())
med = media(notaA, notaB, notaC)
print "MEDIA = {0:.1f}".format(med)
# print ("MEDIA = {0:.1f}".format(med))